django_declarative_apis/machinery/filtering.py

```diff
@@ -144,18 +174,17 @@ def _make_filter_cache_key(expand_children, klass, pk):
     return (str(expand_children), klass, str(pk))
 
 
+def _make_model_cache_key(klass, pk):
+    return (klass, str(pk))
```

Comment:
what will this key look like if we have to manually inspect it in cache? Can we have a more human readable cache key here, maybe something like `{klass.__cache__.__name}:{str(pk)}`

Reply:
nevermind, i see it matches the style of the cache key of the other cache